lib/image-diff.js

```diff
@@ -54,17 +54,20 @@ ImageDiff.createDiff = function (options, cb) {
   // http://www.imagemagick.org/script/compare.php
   var diffCmd = 'compare';
   var diffArgs = [
-    'compare',
     '-verbose',
     // TODO: metric and highlight could become constructor options
     '-metric', 'RMSE',
-    '-highlight-color', 'RED',
-    '-compose', 'Src',
+    '-dissimilarity-threshold', '1',
+    '-highlight-color', 'RED']
+    //shadow options if options.shadow is set
```

Comment:
Please add a space after `//` and use Sentence casing for options to be consistent with other comments:

```js
// Shadow options if options.shadow is set
```